stuff/pipelines/test_producer_contest.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+
+'''test producer of contests.'''
+
+import json
+import dataclasses
+import os
+import sys
+
+from typing import Dict, Any
+import pytest
+import pika
+import pytest_mock 
+
+import credentials
+import rabbitmq_connection
+import producer_contest
+import rabbitmq_client
+import rabbitmq_connection
+
+sys.path.insert(
+    0,
+    os.path.join(
+        os.path.dirname(os.path.dirname(os.path.realpath(__file__))), "."))
+import lib.db   # pylint: disable=wrong-import-position
+
+@dataclasses.dataclass
+class Message:
```

Comment:
```suggestion
class MessageSavingCallback:
```

esto no es un mensaje: es un callback que guarda los mensajes. al momento de elegir nombres de clases, hay que intentar elegir un nombre que describa lo que es la clase y además para lo que se va a usar.